gx_glue_catalog_tutorial/infra/variables.tf

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
Please remove this starting newline and run `terraform fmt` on this file to line up the `=`s